stable/minio/values.yaml

```diff
@@ -54,6 +54,17 @@ resources:
     memory: 256Mi
     cpu: 250m
 
+## Create a bucket after minio install
```

Comment:
@yagonobre Let's change this documentation to be consistent with the rest:

```yaml
## Create a bucket after minio install
##
defaultBucket:
  enabled: false
  ## If enabled, must be a string with length > 0
  # name: example
  ## Can be one of none|download|upload|public
  # policy: download
```

If there is a direct reference to these options, you may also want to add a `## ref: LINK` (I'm not familiar enough with Minio yet to know this info, and didn't see it in the docs after a few moments looking in https://docs.minio.io/).

Reply:
I think that dont exist a direct link
@NitishT, have you info about?

Reply:
Default bucket just gives an option to users who want to have a bucket created when they deploy Minio, so there is no doc link for this. I will make a note to add a link here, if we add a doc on docs.minio.io

Reply:
Thanks @NitishT - OK great 👍 

@yagonobre This default in the values file should still be updated to this (which adds documentation as comments, and is more DRY):
```diff
+## Create a bucket after minio install
+##
+defaultBucket:
+  enabled: false
+  ## If enabled, must be a string with length > 0
+  # name: example
+  ## Can be one of none|download|upload|public
+  # policy: download
+
```

Instead of what you have in the PR now (which is redundant):
```diff
+## Create a bucket after minio install
+##defaultBucket
+##  enabled: true
+##  name: bucketName
+##  policy: "none|download|upload|public"
+##
+defaultBucket:
+  enabled: false
+  name:
+  policy: "download"
+
```

Does that make sense?

Reply:
updated :)